extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -0,0 +1,361 @@
+
+			let sourceTargetLabels = view.modelBuilder.flexContainer()
+				.withProperties({
+					alignItems: 'stretch',
+					horizontal: true
+				}).component();
+
+			this.sourceTargetFlexLayout = view.modelBuilder.flexContainer()
+				.withProperties({
+					alignItems: 'stretch',
+					horizontal: true
+				}).component();
+
+			this.createSwitchButton(view);
+			this.createCompareButton(view);
+			this.createGenerateScriptButton(view);
+			this.resetButtons();
+
+			let toolBar = view.modelBuilder.toolbarContainer();
+			toolBar.addToolbarItems([{
+				component: this.compareButton
+			}, {
+				component: this.generateScriptButton,
+				toolbarSeparatorAfter: true
+			},
+			{
+				component: this.switchButton
+			}]);
+
+			let sourceLabel = view.modelBuilder.text().withProperties({
+				value: localize('schemaCompare.sourceLabel', 'Source')
+			}).component();
+
+			let targetLabel = view.modelBuilder.text().withProperties({
+				value: localize('schemaCompare.targetLabel', 'Target')
+			}).component();
+
+			let arrowLabel = view.modelBuilder.text().withProperties({
+				value: localize('schemaCompare.switchLabel', '➔')
+			}).component();
+
+			this.sourceNameComponent = view.modelBuilder.table().withProperties({
+				columns: [
+					{
+						value: sourceName,
+						headerCssClass: 'no-borders',
+						toolTip: sourceName
+					},
+				]
+			}).component();
+
+			this.targetNameComponent = view.modelBuilder.table().withProperties({
+				columns: [
+					{
+						value: targetName,
+						headerCssClass: 'no-borders',
+						toolTip: targetName
+					},
+				]
+			}).component();
+
+			sourceTargetLabels.addItem(sourceLabel, { CSSStyles: { 'width': '55%', 'margin-left': '15px', 'font-size': 'larger', 'font-weight': 'bold' } });
+			sourceTargetLabels.addItem(targetLabel, { CSSStyles: { 'width': '45%', 'font-size': 'larger', 'font-weight': 'bold' } });
+			this.sourceTargetFlexLayout.addItem(this.sourceNameComponent, { CSSStyles: { 'width': '45%', 'height': '25px', 'margin-top': '10px', 'margin-left': '15px' } });
+			this.sourceTargetFlexLayout.addItem(arrowLabel, { CSSStyles: { 'width': '10%', 'font-size': 'larger', 'text-align-last': 'center' } });
+			this.sourceTargetFlexLayout.addItem(this.targetNameComponent, { CSSStyles: { 'width': '45%', 'height': '25px', 'margin-top': '10px', 'margin-left': '15px' } });
+
+			this.loader = view.modelBuilder.loadingComponent().component();
+			this.noDifferencesLabel = view.modelBuilder.text().withProperties({
+				value: localize('schemaCompare.noDifferences', 'No schema differences were found')
+			}).component();
+
+			this.flexModel = view.modelBuilder.flexContainer().component();
+			this.flexModel.addItem(toolBar.component(), { flex: 'none' });
+			this.flexModel.addItem(sourceTargetLabels, { flex: 'none' });
+			this.flexModel.addItem(this.sourceTargetFlexLayout, { flex: 'none' });
+			this.flexModel.addItem(this.loader, { CSSStyles: { 'margin-top': '2em' } });
```

Comment:
>em [](start = 70, length = 2)

Any particular reason you're using em here and px everywhere else?

Reply:
nope. I changed it to px to be consistent with everywhere else